lib/reporting/report_renderer.rb

```diff
@@ -10,7 +10,13 @@ def initialize(report)
       @report = report
     end
 
+    # Strip headers and footers for these formats
```

Comment:
Does footers refer to the summary rows?

Reply:
if `raw_render?` is only used to decide about the header and summary.  Maybe we can use a better name?

Reply:
Yes, you're right it should say "summary rows" rather than "footers".

I agree it would be good to have a better name. I'm not sure what is better though. Maybe `render_data_only?`

Reply:
I think this is better :)

Reply:
I've updated the comment, but opted to avoid renaming the method because I hope to remove it anyway.